File-Formats/DocIO/Create-Word-document-in-UWP.md

```diff
@@ -14,11 +14,11 @@ Syncfusion Essential DocIO is a [UWP Word library](https://www.syncfusion.com/do
 
 Step 1: Create a new C# Blank App (Universal Windows) project.
 
-![Create UWP application in Visual Studio](UWP_images/CreateProject.png)
+![Create UWP application in Visual Studio](uwp_images/create-project-in-uwp-word.png)
```

Comment:
Change like as below,
create-uwp-project is enough.

Reply:
modified